templates/.travis.yml.mustache

```diff
@@ -0,0 +1,45 @@
+language: nix
+
+script:
+- nix-build --argstr coq-version-or-url "$COQ" --extra-substituters https://coq.cachix.org --trusted-public-keys "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= coq.cachix.org-1:Jgt0DwGAUo+wpxCM52k2V+E0hLoOzFPzvg94F65agtI="
+
+matrix:
+  include:
+
+  # Test supported versions of Coq
+{{# tested_coq_versions }}
+  - env: COQ={{ version_or_url }}
+{{/ tested_coq_versions }}
+
+  # Test opam package
+  - language: minimal
+    sudo: required
+    services: docker
+    env:
+    - COQ_IMAGE=coqorg/coq:{{ tested_coq_opam_version }}
+    - CONTRIB_NAME={{ shortname }}
+    - NJOBS=2
+    install: |
+      # Prepare the COQ container
+      docker run -d -i --init --name=COQ -v ${TRAVIS_BUILD_DIR}:/home/coq/${CONTRIB_NAME} -w /home/coq/${CONTRIB_NAME} ${COQ_IMAGE}
+      docker exec COQ /bin/bash --login -c "
+        # This bash script is double-quoted to interpolate Travis CI env vars:
+        echo \"Build triggered by ${TRAVIS_EVENT_TYPE}\"
+        export PS4='+ \e[33;1m(\$0 @ line \$LINENO) \$\e[0m '
+        set -ex  # -e = exit on failure; -x = trace for debug
+        opam update -y
```

Comment:
@Zimmi48 note that the current version of this `docker exec` command only prints metadata (`opam config list`, etc.).
So it could be useful to add the following line, just after `opam update -y`?
```
opam install -y -j ${NJOBS} --deps-only .
```
This way, it would follow Travis CI's standard separation between `install:` commands for dependencies, and `script:` commands…
Apart from that, nice PR!